common/rpc/interceptor/telemetry.go

```diff
@@ -292,7 +292,13 @@ func (ti *TelemetryInterceptor) handleError(
 	//	*serviceerror.Unavailable:
 	default:
 		metricsHandler.Counter(metrics.ServiceFailures.GetMetricName()).Record(1)
-		ti.logger.Error("service failures", append(logTags, tag.Error(err))...)
+
+		// TODO: do this with a proper type comparison
+		skipLog := err.Error() == "Shutting down"
```

Comment:
Another option here is to just do this check at the top of the function, where it checks for context canceled, etc. I thought it would be good to include the metrics and just skip the logs, but if we're shutting down we probably won't collect the metrics anyway, right?